pex/interpreter.py

```diff
@@ -162,7 +172,13 @@ def platform_tag(self):
 
     @property
     def version(self):
-        return self._version
+        # type: () -> Tuple[int, int, int]
+        """The interpreter version as a normalized tuple.
+
+        This assumes that the interpreter uses semantic versioning in the form
+        <major>.<minor>.<patch>, which is used by CPython, PyPy, and Jython.
```

Comment:
Please let me know if this is not safe. The only interpreter I can think of is `IronPython`, which also uses semvar it looks like: https://ironpython.net.

Reply:
Thanks for pointing out `sys.version_info`! Will do.